### DISCUSSION TITLE
Custom Homepage: Help with Customizing Homepage to Remove Default Features and Add a Custom Counter

### DISCUSSION THREAD

**Discussion by ghost** [nunocoracao/blowfish#1861] [Q&A]
Hello again,

I hope I can express myself clearly on this issue. 

I want to customize my homepage by removing the default suggested features (like the logo/author picture, heading, etc.) and replacing them with a custom counter that I’ve compiled using ChatGPT. The counter consists of HTML, JavaScript, and CSS.

Specifically, I’d like to replace the default content/container? (such as the logo, author name, heading, and bio) in the center of the page with my own custom counter.

Can anyone guide me on how to make these changes? I would really appreciate any assistance with removing the default elements and integrating my custom counter into the homepage.

I have no experience in coding.
Thank you!

**Comment by chang48** [comment]
I'm new to blowfish, but as a general advice, I found that leveraging the power of Hugo's project structure as mentioned [here](https://blowfish.page/docs/advanced-customisation/) is an excellent approach. Also: [Custom layouts](https://blowfish.page/docs/content-examples/#custom-layouts).

Given the following generic Hugo site structure:
```
my-site/
├── archetypes/
├── assets/
├── content/
├── data/
├── i18n/
├── layouts/
├── static/
├── themes/
```
you could simply copy the configurations in [exampleSite](https://github.com/nunocoracao/blowfish/tree/main/exampleSite) to corresponding project folders and start customizing. Any modifications you do will override blowfish's defaults.

Also consider using `_index.md`. The reason is explained [here](https://gohugo.io/content-management/organization/#index-pages-_indexmd). However, the customization you are after is pretty involved. It is beyond my ability to answer. Good luck with your endeavors.
